iceoryx_hoofs/source/cxx/requires.cpp

```diff
@@ -29,7 +30,8 @@ namespace internal
 /// @NOLINTJUSTIFICATION todo #1196 will be obsolote with new error handler
 /// @NOLINTNEXTLINE(readability-function-size)
 void Require(
-    const bool condition, const char* file, const int line, const char* function, const char* conditionString) noexcept
+    const bool condition, const char* file, const int32_t line, const char* function, const char* conditionString)
```

Comment:
I think similar to `file` being a `const char*`, `line` should remain an `int` since it is meant to be used with the compiler defined macro `__LINE__` which is also an `int`